### PR TITLE
arm64: dts: qcom: msm8953: typo fix for modem pll supply

### PR DESCRIPTION
**Pull request by M0Rf30**
this will avoid the usage of dummy regulator for the modem remoteproc and avoid a warning message within the dmesg

### CODE REVIEW

File: arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts [hunk 1]
```diff
@@ -301,8 +301,9 @@
 &modem {
 	status = "okay";
 
+	firmware-name = "qcom/sdm625/mba.mbn", "qcom/sdm625/modem.mdt";
```
Comment (z3ntu) [inline arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts:304]: This is an unrelated change, so at least make a separate commit from this.

Comment (fekz115) [reply arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts:304]: Maybe just remove this line? I think if it required, it should be present in all the dts, but it is just firmware path

Comment (M0Rf30) [reply arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts]: Yeah a mistake. cherry-picking the commit from another branch of mine
